### DISCUSSION TITLE
redirecting user after registration

### DISCUSSION THREAD

**Discussion by Moemen12** [clerk/javascript#3824] [Support request - DO NOT OPEN HERE]
hello guys im facing a problem with the latest clerk version , when user try to signup using google it works , but the problem after signup and redirecting to / route (http://localhost:3000/) it redirects me from / to https://rested-wombat-61.accounts.dev/sign-up , i need the redirection to be only to http://localhost:3000/

i tried many things in .env.local file like but they didnt work, 
notice : signup functionality works my problem only with the second redirection that should not happen

```
NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=
CLERK_SECRET_KEY=

# NEXT_PUBLIC_CLERK_SIGN_IN_FALLBACK_REDIRECT_URL=/
# NEXT_PUBLIC_CLERK_SIGN_UP_FALLBACK_REDIRECT_URL=/

NEXT_PUBLIC_CLERK_SIGN_IN_FORCE_REDIRECT_URL=/
NEXT_PUBLIC_CLERK_SIGN_UP_FORCE_REDIRECT_URL=/

NEXT_PUBLIC_CLERK_SIGN_IN_URL=/sign-in
NEXT_PUBLIC_CLERK_SIGN_UP_URL=/sign-up



```

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829

**Comment by ycocerious** [comment]
Were you able to fix this issue?
Facing a similar issue here

**Reply by Moemen12** [reply]
No actually

**Reply by ycocerious** [reply]
I have asked their support in discord
waiting on an answer from them
will share if i find a solution

**Reply by ycocerious** [reply]
After a bunch of debugging, realised that this was being caused due to the <ClerkProvider appearance = {{baseTheme: dark}}> in my codebase
Removed the global theming and gave it individually to each clerk component and it's working (although the theme isnt applied when user clicks on "manage account")
